infra/bridge/src/controllers/phone.js

```diff
@@ -149,7 +149,9 @@ router.post('/verify', phoneVerifyCode, async (req, res) => {
   const attestation = await generateAttestation(
     AttestationTypes.PHONE,
     attestationBody,
-    attestationValue,
+    {
+      uniqueId: attestationValue
```

Comment:
what is the value here?

Reply:
It's normalized phone number. https://github.com/OriginProtocol/origin/blob/b2be87875ee7764e2b922fa21cc762bd9b5e3754/infra/bridge/src/controllers/phone.js#L140-L147